gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -37,27 +37,30 @@
 import datetime
 import enum
 import logging
-import re
 import tempfile
 
 import apache_beam as beam
 from apache_beam.io.filesystems import FileSystems
-from apache_beam.io.gcp.internal.clients import bigquery
-from apache_beam.options.pipeline_options import PipelineOptions
 from apache_beam.options.pipeline_options import GoogleCloudOptions
-from apitools.base.py import exceptions
-from oauth2client.client import GoogleCredentials
+from apache_beam.options.pipeline_options import PipelineOptions
 
-# TODO: Replace with the version from Beam SDK once that is released.
-from gcp_variant_transforms.beam_io import vcfio
 from gcp_variant_transforms.beam_io import vcf_header_io
+from gcp_variant_transforms.beam_io import vcfio
 from gcp_variant_transforms.libs import vcf_header_parser
-from gcp_variant_transforms.libs.variant_merge import move_to_calls_strategy
 from gcp_variant_transforms.libs.variant_merge import merge_with_non_variants_strategy
+from gcp_variant_transforms.libs.variant_merge import move_to_calls_strategy
+from gcp_variant_transforms.options import variant_transform_options
 from gcp_variant_transforms.transforms import filter_variants
+from gcp_variant_transforms.transforms import merge_headers
 from gcp_variant_transforms.transforms import merge_variants
 from gcp_variant_transforms.transforms import variant_to_bigquery
-from gcp_variant_transforms.transforms import merge_headers
+
+OPTIONS = [
+    variant_transform_options.VcfReadOptions(),
```

Comment:
I think it's not recommended to initialize classes in module level variables (IIRC, there is even a styleguide violation about this b/c of the way assignment happens for these variables). Consider just setting the module name "variant_transform_options.VcfReadOptions" and initializing it in run (e.g. you can do command_line_options = [option() for option in _COMMAND_LINE_OPTIONS] and then pass that list to your methods).

Reply:
Done.